site/src/pages/ManagementSettingsPage/CustomRolesPage/CreateEditRolePageView.tsx

```diff
@@ -323,7 +412,12 @@ const styles = {
 	}),
 	actionDescription: (theme) => ({
 		color: theme.palette.text.secondary,
+		paddingTop: 6,
 	}),
+	actionItem: {
+		display: "grid",
+		gridTemplateColumns: "270px 3fr",
```

Comment:
ultra nit: this could be `1fr` since there aren't any other columns taking fractions